src/services

```diff
@@ -1,11 +1,13 @@
 p({"class":"foam.nanos.boot.NSpec", "name":"logMessageDAO", "lazy":true, "serve":true, "serviceScript":"new foam.dao.EasyDAO.Builder(x).setOf(foam.nanos.logger.LogMessage.getOwnClassInfo()).setInnerDAO(((foam.dao.DAO) x.get(\"localLogMessageDAO\"))).build();", "client":"{\"of\":\"foam.nanos.logger.LogMessage\"}"})
 p({"class":"foam.nanos.boot.NSpec", "name":"logLevelFilterLogger",             "lazy":false, "serve":false, "service":{"class":"foam.nanos.logger.LogLevelFilterLogger"}})
 p({"class":"foam.nanos.boot.NSpec", "name":"logger",                           "lazy":false, "serve":false, "serviceScript":"import foam.nanos.logger.CompositeLogger; import foam.nanos.logger.StdoutLogger; import foam.nanos.logger.FileLogger; import foam.nanos.logger.ProxyLogger; import foam.nanos.logger.LogLevelFilterLogger; import foam.nanos.logger.DAOLogger; CompositeLogger log = new CompositeLogger(); StdoutLogger stdOut = new StdoutLogger(); ProxyLogger logLevelFilterLogger = (ProxyLogger) x.get(\"logLevelFilterLogger\"); logLevelFilterLogger.setX(x); logLevelFilterLogger.setDelegate(log); DAOLogger daoLogger = new DAOLogger(x); log.add(stdOut).add(daoLogger); return logLevelFilterLogger;"})
 
-p({"class":"foam.nanos.boot.NSpec", "name":"notificationDAO",                  "lazy":false,  "serve":true,  "serviceScript":"return new foam.dao.PMDAO(x, new foam.nanos.notification.NotificationTemplateDAO.Builder(x).setDelegate( new foam.dao.SequenceNumberDAO( new foam.nanos.notification.SendNotificationDAO(x, new foam.nanos.notification.SendEmailNotificationDAO(new foam.nanos.notification.SendSlackNotificationDAO(new foam.dao.java.JDAO(x, foam.nanos.notification.Notification.getOwnClassInfo(), \"notifications\")))))).build());", "client":"{\"of\":\"foam.nanos.notification.Notification\"}"})
+p({"class":"foam.nanos.boot.NSpec", "name":"localNotificationDAO", "lazy":false, "serve":false, "serviceScript":"return  new foam.dao.EasyDAO.Builder(x).setOf(foam.nanos.notification.Notification.getOwnClassInfo()).setIndex(new foam.core.PropertyInfo[] {foam.nanos.notification.Notification.GROUP_ID}).setAuthenticate(false).setPm(true).setSeqNo(true).setDecorator(new foam.nanos.notification.NotificationTemplateDAO.Builder(x).setDelegate(new foam.nanos.notification.SendNotificationDAO(x, new foam.nanos.notification.SendEmailNotificationDAO(new foam.nanos.notification.SendSlackNotificationDAO(x)))).build()).setJournalType(foam.dao.JournalType.SINGLE_JOURNAL).setJournalName(\"notifications\").build();"})
```

Comment:
`authenticate` here still would be true because that is the default for NSpec. I don't think we should set `authenticate` on EasyDAO as it is going to be removed. Authentication is handled on service level and should be set on NSpec